release notes/v0.38.0.md

```diff
@@ -0,0 +1,275 @@
+
+Here's an example of using the `S3Client` to download some input from S3, and upload results back to it:
+```javascript
+import { check } from 'k6';
+import exec from 'k6/execution';
+import http from 'k6/http';
+import { AWSConfig, S3Client } from 'https://jslib.k6.io/aws/0.4.0/s3.js';
+
+const awsConfig = new AWSConfig(
+  __ENV.AWS_REGION,
+  __ENV.AWS_ACCESS_KEY_ID,
+  __ENV.AWS_SECRET_ACCESS_KEY
+);
+
+const s3 = new S3Client(awsConfig);
+const testBucketName = 'test-jslib-aws';
+const testInputFileKey = 'productIDs.json';
+const testOutputFileKey = `results-${Date.now()}.json`;
+
+export function setup() {
+  // If our test bucket does not exist, abort the execution.
+  const buckets = s3.listBuckets();
+  if (buckets.filter((b) => b.name === testBucketName).length == 0) {
+    throw `bucket ${testBucketName} does not exist`
+  }
+  
+  // If our test object does not exist, abort the execution.
+  const objects = s3.listObjects(testBucketName);
+  if (objects.filter((o) => o.key === testInputFileKey).length == 0) {
+    throw `object ${testInputFileKey} not found in bucket ${testBucketName}`
+  }
+
+  // Download the S3 object containing our test data
+  const inputObject = s3.getObject(testBucketName, testInputFileKey);
+  
+  // Let's return the downloaded S3 object's data from the
+  // setup function to allow the default function to use it.
+  return {
+    productIDs: JSON.parse(inputObject.data),
+  };
+}
+
+export default function (data) {
+  // Pick a random product ID from our test data
+  const randomProductID = data.productIDs[Math.floor(Math.random() * data.productIDs.length)];
+  // Query our ecommerce website's product page using the ID
+  const res = http.get(`http://your.website.com/product/${randomProductID}/`);
+  check(res, { 'is status 200': res.status === 200 });
+}
+
+export function handleSummary(data) {
+  // Once the load test is over, let's upload the results to our
+  // S3 bucket. This is executed after teardown.
+  s3.putObject(testBucketName, testOutputFileKey, JSON.stringify(data));
+}
+```
+
+Head to the [documentation](https://k6.io/docs/javascript-api/jslib/aws) for more examples and details on how to use the library in your own scripts.
+
+
+## Breaking Changes
+
+### `console.log` and `console.info` pretty prints objects and arrays ([2375](https://github.com/grafana/k6/pull/2375))
+
+For convenience, methods like `console.log` and `console.info` will now automatically `JSON.stringify` objects and arrays passed to it. Thus, when the `console.log({'foo': 'bar'})` statement used to print `[object Object]`, it will now print `{'foo': 'bar'}` instead, which will make the experience of debugging k6 scripts easier and more intuitive. 
```

Comment:
```suggestion
For convenience, all `console` methods such as `console.log` and `console.info` will now automatically `JSON.stringify` objects and arrays passed to it. Thus, instead of `console.log({'foo': 'bar'})` printing `[object Object]`, it will now print `{'foo': 'bar'}`, which will make the experience of debugging k6 scripts easier and more intuitive. 
```